content/12-truthy-and-falsy-values-in-javascript/post.md

```diff
@@ -0,0 +1,77 @@
+Title: »Truthy« and »falsy« values in JavaScript
+
+-----
+
+Date: 1428522382
+
+-----
+
+Contributors: rasshofer
+
+-----
+
+Text:
+
+The terms »truthy« and »falsy« are an essential feature (not a bug!) of the JavaScript language, stating that values aren’t just `true` when talking about »truthy« neither just `false` when talking about »falsy«. What’s meant is that values coerce to `true` or `false` when evaluated in a boolean context. Thus, something which evaluates to `true` is truthy and something which evaluates to `false` is falsey. 
```

Comment:
»falsey« looks funny. It think neither »falsy« nor »truthy« or »thruthey« are real words. However, I heard the ECMAScript docs use »truthy« and »falsy« sometimes, maybe that are the terms to use.